flybywiresim/aircraft
# feat(a380x): Update Initial Fuel Levels

Fixes #9321 

## Summary of Changes

Changes to the starting fuel in all tanks to better represent a real life starting load. 
Fuel is down from ~180 tonne to 15 tonne, allowing more people to fuel the plane rather than defuel the plane during ground ops. 
Tank balance was taken from replicating the load within the EFB in current release. 

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Yahtzee94

## Testing instructions

1. Open aircraft on ramp or runway
2. Verify in EFB and SD that fuel is total to 15000kg 
3. Verify in EFB and SD that fuel is 3750kgs in feed tanks 1 through 4 


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
